hydra-node/test/Hydra/API/ServerSpec.hs

```diff
@@ -95,25 +102,87 @@ spec = parallel $ do
       monitor $ cover 0.1 (null outputs) "no message when reconnecting"
       monitor $ cover 0.1 (length outputs == 1) "only one message when reconnecting"
       monitor $ cover 1 (length outputs > 1) "more than one message when reconnecting"
-      run . failAfter 5 $ do
+      run . failAfter 15 $ do
```

Comment:
I would add an explanation, at least in a commit message, as to why we need to replace 5 by 15 here.

Reply:
Ooops this must be leftover. I'll revert.